docs/permissions.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Permissions
+---
+
+Administrate's base logic assumes any signed-in super-user
+can access any record and run any operation.
+You can manage permissions by placing logic in your controllers.
+
+[Pundit] is a popular gem many people describe permissions rules in.
+Administrate's Pundit plugin applies your permissions across your dashboards.
+See the end of this page.
+
+## Basic permissions
+
+Permissions can be described in `Admin::ApplicationController`
+or any subclass.
+
+Here are some examples:
```

Comment:
I think this doesn't make it clear enough that the examples are indeed the full API that users will need to override in order to implement authorization. How about the following?

```suggestion
The following are the methods that you will need to override in order to implement your own permission scheme, shown with example implementations:
```